### PR TITLE
Added autologin and guest login feauture

### LINKED ISSUE #78: Add support for guest accounts and automatic login
On `settings.sh` add also these options:
```
GUEST_ACCOUNTS="yes"
AUTOMATIC_LOGIN="yes"
```

When `GUEST_ACCOUNT` is "yes", the LTSP server should support guest account logins from the clients (see this blog which explains what this means and why it can be useful: http://dashohoxha.fs.al/linux-guest-account/)

When `AUTOMATIC_LOGIN` is "yes", then the client should login automatically to the guest account as soon as it boots up.

I think that these can be implemented easily by small changes on the configuration of LTSP, however I don't know what these changes are. Maybe we can ask help from the LTSP community about these.

### PR DESCRIPTION
**Pull request by d78ui98**
closes #78

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -68,3 +68,17 @@ fi
...
+        -e "/^#LDM_AUTOLOGIN/ a LDM_USERNAME=vagrant \nLDM_PASSWORD=vagrant" \
+        -e "/^#LDM_AUTOLOGIN/ c LDM_AUTOLOGIN=True"
+fi 
+
+# Provide Login as Guest button to directly login from client
+if [[ ${GUEST_ACCOUNT,,} == "yes" ]]; then
+    sed -i /var/lib/tftpboot/ltsp/i386/lts.conf \
+        -e "/^#LDM_GUESTLOGIN/ a LDM_USERNAME=vagrant \nLDM_PASSWORD=vagrant" \
+        -e "/^#LDM_GUESTLOGIN/ c LDM_GUESTLOGIN=True"
+fi
```
Comment (akash0x53) [inline install.sh]: Any service need a restart  after changing config?

File: install.sh [hunk 2]
```diff
@@ -68,3 +68,17 @@ fi
 
 # Restarting service
 service dnsmasq restart
+
+# Automatically login from clients 
+if [[ ${AUTOMATIC_LOGIN,,} == "yes" ]]; then
+    sed -i /var/lib/tftpboot/ltsp/i386/lts.conf \
```
Comment (akash0x53) [inline install.sh]: Please check `ltsp-config` output, it give you which file it had created and use it here.

Comment (akash0x53) [reply install.sh]: For example
```
akash@skyfall:/var/lib/tftpboot/ltsp$ sudo ltsp-config -o lts.conf|cut -d' ' -f2
/var/lib/tftpboot/ltsp/amd64/lts.conf

```

File: install.sh (hunk 1 above)
Comment (d78ui98) [reply install.sh]: No. Not at all. We just need to restart client.

File: install.sh (hunk 2 above)
Comment (d78ui98) [reply install.sh]: please correct me if I am wrong. 
Are we doing this to not hardcode the `lts.conf` path?

Comment (akash0x53) [reply install.sh]: Right. `ltsp-config` returns lts.conf path

Comment (d78ui98) [reply install.sh]: Also it will handle both cases `i386` and `amd64`
clever :+1:

**Review by akash0x53 — APPROVED** [review]
Looks good

**Pull request merged**